src/index.ts

```diff
@@ -36,6 +37,10 @@ export default class I18n {
    * Retrieves a key from the translations object.
    */
   getKey (path: string): TranslationObject | string {
+    if (isEmpty(this.translations)) {
+      throw new Error('Translations have not been loaded')
+    }
```

Comment:
should this be triggered lazily on in `setTranslations`?

Reply:
I don't get what you mean. What I want to ensure with this is that `setTranslations`  was not called.

Reply:
got it 👌